### PR TITLE
editorOptions to disable the picker, fields or preset colors

### PR DESCRIPTION
**Pull request by pmaas**
Added editorOptions to disable the picker, hex/rgba fields or preset colors:

```
editor: 'Shel.Neos.ColorPicker/ColorPickerEditor'
editorOptions:
  picker: bool
  fields: bool
  presetColors: array || bool
```

**Examples:**

`fields: false` and `presetColors: false`:
Bildschirmfoto 2019-07-31 um 23 06 28

`picker: false` and `presetColors: false`:
Bildschirmfoto 2019-07-31 um 23 07 50

`picker: false` and `fields: false`:
Bildschirmfoto 2019-07-31 um 23 05 46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,12 +40,16 @@ Add a property of type string and configure the editor as seen in this example:
...
+              editorOptions:
+                disablePicker: false        # optional
+                disableFields: false        # optional
+                disablePresetColors: false  # optional
               
 ## Customization
 
 The editor allows some customization options via your `Settings.yaml` file:
-
+es
```
Comment (Sebobo) [inline README.md]: This breaks the output ;)

File: README.md [hunk 2]
```diff
@@ -40,12 +40,16 @@ Add a property of type string and configure the editor as seen in this example:
             reloadIfChanged: true
             inspector:
               group: 'text'
-              editor: 'Shel.Neos.ColorPicker/ColorPickerEditor'   
+              editor: 'Shel.Neos.ColorPicker/ColorPickerEditor'
+              editorOptions:
+                disablePicker: false        # optional
```
Comment (Sebobo) [inline README.md]: As discussed in Slack I would also think we don't need the `disable` prefix and invert the option behaviour to have shorter options.

**Comment by Sebobo** [comment]
And of course thanks a lot again for the great contribution :)

File: README.md (hunk 1 above)
Comment (pmaas) [reply README.md]: Uh, was a little bit to late ;) Fixed.

File: README.md (hunk 2 above)
Comment (pmaas) [reply README.md]: Renamed the `editorOptions` to:

```
editorOptions:
  picker: bool
  fields: bool
  presetColors: array || bool
```

**Comment by pmaas** [comment]
> Please adapt the option names and fix the `Settings.yaml`

Done :)

**Comment by jonnitto** [comment]
This is a massive improvement, thank you @pmaas !

**Pull request merged**